api/replication.storage/v1alpha1/volumegroupreplication_types.go

```diff
@@ -34,8 +34,7 @@ type VolumeGroupReplicationSpec struct {
 
 	// volumeReplicationClassName is the volumeReplicationClass name for the VolumeReplication object
 	// created for this volumeGroupReplication
-	// +kubebuilder:validation:Required
-	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="volumeReplicationClassName is immutable"
+	// +kubebuilder:validation:Optional
```

Comment:
We can have immutability built into the field, need not lose that. Just the field needs to be optional and not mandatory.